packages/vite/src/node/plugins/css.ts

```diff
@@ -404,6 +427,9 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
         map,
       }
     },
+    buildEnd() {
+      preprocessorWorkerController.close()
+    },
```

Comment:
nit: Would be nice to move this above and below `buildStart` so it's consistent with the other plugins. I think it's nice to see the startup and teardown closer together too.